flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/source/SplitHelpers.java

```diff
@@ -72,7 +73,8 @@ private SplitHelpers() {
       }
 
       final ScanContext scanContext = ScanContext.builder().build();
-      final List<IcebergSourceSplit> splits = FlinkSplitPlanner.planIcebergSourceSplits(table, scanContext);
+      final List<IcebergSourceSplit> splits = FlinkSplitPlanner.planIcebergSourceSplits(
+          table, scanContext, ThreadPools.getWorkerPool());
```

Comment:
this uses the default/shared worker pool, while others use a separate pool. what are the reasons?

Reply:
nm. I guess it is because this is a test class.